DnD-Montreal/session-tome
# Add Character + Entry stubbing for DnD Beyond import

## Description 
Closes #69 nice.

Adds an endpoint that accepts a `beyond_link` of format "https://www.dndbeyond.com/anything/123456". Endpoint calls Beyond adapter and extracts a character from the API, and then stubs out the appropriate amount of entries for the character given their level

## How to Test
1. Gather a link for a public DnD Beyond character (https://www.dndbeyond.com/characters/5441197)
2. Authenticate as a user in the database, (theres a little weirdness with the Auth via REST clients im wrestling with so ping me for details on that)
3. Using a REST client, like Insomnia, call the endpoint `/beyond-import`
   - ensure the request body has a `beyond_link` parameter
4. Observe that the endpoint returns a character named "Donte Greyson", a level 20 fighter
5. Observe that that same character has 20 log sheet entries
